src/cli/project.py

```diff
@@ -186,19 +186,19 @@ def use(ctx, project_id, remove, **kwargs):
         console.exit_generic_error()
 
     project_list = data["allProjects"]["results"]
-    project_dict = {project["id"]: project["title"] for project in project_list}
+    project_dict = {project["id"]: project for project in project_list}
 
     # argument
     if not project_id:
         project_title = console.list(
             message="Please select a project",
-            choices=project_dict.values(),
+            choices=[project["title"] for project in project_dict.values()],
         )
         if project_title is None:
             return False
 
-        for id, title in project_dict.items():
-            if title == project_title:
+        for id, project in project_dict.items():
+            if project["title"] == project_title:
```

Comment:
Wouldn't it be better to have the `id` instead of the `title` as selecting parameter? We do have that in many places in our code base.

Reply:
Yes, the ID should be the selecting parameter within the CLI. However, when using the `inquirer.List` as interactive selection option, we display the title/names. These titles have to be mapped back to IDs.

I opened #30.